examples/src/app/helpers/strings.mjs

```diff
@@ -0,0 +1,74 @@
+/**
+ * @example
+ * capitalizeFirstLetter("test") // Outputs 'Test'
+ * @param {string} string 
+ * @returns {string}
+ */
+function capitalizeFirstLetter(string) {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+/**
+ * @example
+ * toKebabCase('BlendTrees1D'); // Outputs: 'blend-trees-1d'
+ * toKebabCase('LightsBakedAO'); // Outputs 'lights-baked-a-o'
+ * @param {string} str - The string.
+ * @returns String in kebab-case-format
+ */
+function toKebabCase(str) {
```

Comment:
I'd keep it simple, perhaps a solution 2 from here: https://www.geeksforgeeks.org/how-to-convert-a-string-into-kebab-case-using-javascript/
we don't need AO to be -a-o, it's fine to be -ao
and also 1d does not need to be -1d but can be -1-d

same goes for other functions in this file .. if there's a simpler way, I'm up for it.

Reply:
yep sure, just be aware that if there's a simpler way to do all this, even if the end strings are different than in current examples, feel free to use it.